test/utils/navigator.py

```diff
@@ -81,13 +82,15 @@ class TezosNavigator(metaclass=MetaScreen):
 
     use_case_home       = UseCaseHome
     use_case_settings   = UseCaseSettings
+    use_case_sub_settings = UseCaseSubSettings
```

Comment:
You can use the already defined `settings` instead of reintroducing a new one.

Reply:
Previous settings and sub_settings had different layout. So exit page and next previous was different. But now it seems they have same layout after the 3.19.0 upgrade.

Reply:
See following in ragger 
```
    "UseCaseSettings": {
        "single_page_exit": BUTTON_LOWER_MIDDLE,
        "multi_page_exit": BUTTON_LOWER_LEFT,
        "previous": BUTTON_LOWER_MIDDLE,
        "next": BUTTON_LOWER_RIGHT,
    },
    "UseCaseSubSettings": {
        "exit": BUTTON_UPPER_LEFT,
        "previous": BUTTON_LOWER_LEFT,
        "next": BUTTON_LOWER_RIGHT,
    },
```

Reply:
Oh yes, you're right !
Now that we have the right version, we can use the `UseCaseSettings`:
https://github.com/LedgerHQ/ragger/blob/master/src/ragger/firmware/stax/positions.py#L206-L216